datashader/tests/test_transfer_functions.py

```diff
@@ -28,14 +29,22 @@ def build_agg(array_module=np):
     return agg
 
 
+def build_agg_dask():
+    return build_agg(np).chunk({d: 1 for d in dims})
+
+
+def create_dask_array(*args, **kwargs):
+    return da.from_array(np.array(*args, **kwargs))
+
+
 try:
     import cupy
-    aggs = [build_agg(np), build_agg(cupy)]
-    arrays = [np.array, cupy.array]
-except ImportError:
+    aggs = [build_agg(np), build_agg(cupy), build_agg_dask()]
+    arrays = [np.array, cupy.array, create_dask_array]
+except (ImportError, ModuleNotFoundError):
```

Comment:
```suggestion
except ImportError:
```

Thanks! This PR looks beautiful. It's failing a linting test ("./datashader/tests/test_transfer_functions.py:44:22: F821 undefined name 'ModuleNotFoundError'"), but that seems to be just that the linter is confused, because ordinarily exceptions don't need to be imported explicitly.

In any case, is there some reason for testing against ModuleNotFoundError specifically? According to https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError, ModuleNotFoundError is a subclass of ImportError and thus should already be caught by checking for ImportError. So unless I'm confused, the PR should be ready to merge if you just apply the suggestion here to revert that change. But I'll leave it for you to apply the suggestion, in case I'm confused about it.

Reply:
While debugging I noticed that a `ModuleNotFoundError` was thrown when importing cupy. However, you are right, it seems not be needed and I updated the PR to revert this.